### PR TITLE
Handle `.` and `./` prefixes

### PR DESCRIPTION
**Pull request by brentleyjones**
Signed-off-by: Brentley Jones <github@brentleyjones.com>

### CODE REVIEW

File: index-import.cpp [hunk 1]
```diff
@@ -68,12 +68,14 @@ struct Remapper {
         // The format() function takes no explicit arguments, instead it gets
         // the values from the match object.
         auto substitution = match.format(replacement);
-        return match.prefix().str() + substitution + match.suffix().str();
+        auto output_str =
+            match.prefix().str() + substitution + match.suffix().str();
+        return path::remove_leading_dotslash(StringRef(output_str)).str();
```
Comment (keith) [inline index-import.cpp:73]: seems like we could probably optimize the swapping between string types here

**Review by keith — APPROVED** [review]

File: index-import.cpp (hunk 1 above)
Comment (brentleyjones) [reply index-import.cpp:73]: I'm open to suggestions. I agree it's ugly.

Comment (keith) [reply index-import.cpp:73]: yea i'd have to look at how the types interact, but I think it's also probably a perf hit, maybe doesn't matter

Comment (keith) [reply index-import.cpp:73]: maybe we'll do later

**Pull request merged**